invesalius/gui/default_viewers.py

```diff
@@ -463,11 +463,12 @@ def OnButtonTarget(self, evt):
         if not self.button_target.IsPressed() and evt is not False:
             self.button_target._pressed = True
             Publisher.sendMessage('Target navigation mode', target_mode=self.button_target._pressed)
-            Publisher.sendMessage('Change camera checkbox', status=self.button_target._pressed)
+            Publisher.sendMessage('Change camera checkbox', status=False)
+            Publisher.sendMessage('Enable volume camera checkbox', status=False)
```

Comment:
This is working fine and is great. I would just recommend changing the name from "Enable" to "Set". Because Enable has a single meaning while the method does both Enable and Disable.

Please, rename the pubsub message to `Set volume camera checkbox`.

Reply:
Thanks, @vhosouza! I totally agree. Done with fd74619